lms/djangoapps/verify_student/views.py

```diff
@@ -504,6 +520,8 @@ def _redirect_if_necessary(
 
         # Redirect if necessary, otherwise implicitly return None
         if url is not None:
+            if waffle.switch_is_active('add-utm-params'):
```

Comment:
where is the code to add this waffle switch, is it added automatically when this code is executed?